modules/shared/components/molecules/Navigation/Navigation.module.css

```diff
@@ -1,32 +1,19 @@
 .navigation-wrapper {
-  @apply p-4 md:px-6 bg-white shadow-navshd flex justify-center;
+  @apply p-4 md:px-6 bg-white md:bg-grey-bg2 shadow-navshd flex justify-center fixed z-20 w-full top-0;
 }
 
 .navigation {
   @apply w-39xl md:w-60xl flex justify-between items-center;
 }
 
-.nav-links-container {
-  @apply flex;
+.nav-links ul {
+  @apply flex items-center;
 }
 
-.brand-name {
-  @apply font-bold text-basev sm:mr-5;
-}
-
-.home-nav-links ul {
-  @apply flex flex-row;
-}
-
-.home-nav-links ul .menu-icon {
-  @apply md:hidden;
-}
-
-.user-nav-links ul {
-  @apply hidden md:flex md:flex-row;
+.nav-links li {
```

Comment:
The responsiveness in the small screens is not perfect as the navbar logo collapse with the navbar icons, so we might consider changing the margin-right value on smaller screens to make it suitable for most screens.
![image](https://user-images.githubusercontent.com/68955535/129029826-b66ddc7c-4e9f-4721-be6c-b310839a61a3.png)

Reply:
Done